notebooks/quickstart.ipynb

```diff
@@ -120,18 +122,115 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "--------------------------------\n",
+    "## The basics of `spaghetti`\n",
+    "### 1. Creating a network instance\n",
+    "\n",
```

Comment:
Add a quick explanation to what a network instance is or why it is important? I think we are missing the 'science story' a bit:

"""Spatial data science techniques can support many types of statistical analysis of spatial networks themselves, and of events that happen along spatial networks in our daily lives, i.e. location of trees along foot paths, biking accidents along street networks or location of coffeeshops along streets. `spaghetti` provides computational tools to support statistical analysis of such events along many different types of networks. Within xyz ..."""

Reply:
👍